cmd/nodecmd/create.go

```diff
@@ -0,0 +1,678 @@
+	inboundGroup := securityGroupBody.AppendNewBlock("ingress", []string{})
+	inboundGroupBody := inboundGroup.Body()
+	inboundGroupBody.SetAttributeValue("description", cty.StringVal("TCP"))
+	inboundGroupBody.SetAttributeValue("from_port", cty.NumberIntVal(22))
+	inboundGroupBody.SetAttributeValue("to_port", cty.NumberIntVal(22))
+	inboundGroupBody.SetAttributeValue("protocol", cty.StringVal("tcp"))
+	var ipList []cty.Value
+	ipList = append(ipList, cty.StringVal(inputIPAddress))
+	inboundGroupBody.SetAttributeValue("cidr_blocks", cty.ListVal(ipList))
+
+	inboundGroup = securityGroupBody.AppendNewBlock("ingress", []string{})
+	inboundGroupBody = inboundGroup.Body()
+	inboundGroupBody.SetAttributeValue("description", cty.StringVal("AVAX HTTP"))
+	inboundGroupBody.SetAttributeValue("from_port", cty.NumberIntVal(9650))
+	inboundGroupBody.SetAttributeValue("to_port", cty.NumberIntVal(9650))
+	inboundGroupBody.SetAttributeValue("protocol", cty.StringVal("tcp"))
+	ipList = []cty.Value{}
+	ipList = append(ipList, cty.StringVal("0.0.0.0/0"))
+	inboundGroupBody.SetAttributeValue("cidr_blocks", cty.ListVal(ipList))
+
+	inboundGroup = securityGroupBody.AppendNewBlock("ingress", []string{})
+	inboundGroupBody = inboundGroup.Body()
+	inboundGroupBody.SetAttributeValue("description", cty.StringVal("AVAX HTTP"))
+	inboundGroupBody.SetAttributeValue("from_port", cty.NumberIntVal(9650))
+	inboundGroupBody.SetAttributeValue("to_port", cty.NumberIntVal(9650))
+	inboundGroupBody.SetAttributeValue("protocol", cty.StringVal("tcp"))
+	ipList = []cty.Value{}
+	ipList = append(ipList, cty.StringVal(inputIPAddress))
+	inboundGroupBody.SetAttributeValue("cidr_blocks", cty.ListVal(ipList))
+
+	inboundGroup = securityGroupBody.AppendNewBlock("ingress", []string{})
+	inboundGroupBody = inboundGroup.Body()
+	inboundGroupBody.SetAttributeValue("description", cty.StringVal("AVAX Staking"))
+	inboundGroupBody.SetAttributeValue("from_port", cty.NumberIntVal(9651))
+	inboundGroupBody.SetAttributeValue("to_port", cty.NumberIntVal(9651))
+	inboundGroupBody.SetAttributeValue("protocol", cty.StringVal("tcp"))
+	ipList = []cty.Value{}
+	ipList = append(ipList, cty.StringVal("0.0.0.0/0"))
+	inboundGroupBody.SetAttributeValue("cidr_blocks", cty.ListVal(ipList))
+
+	inboundGroup = securityGroupBody.AppendNewBlock("egress", []string{})
+	inboundGroupBody = inboundGroup.Body()
+	inboundGroupBody.SetAttributeValue("description", cty.StringVal("Outbound traffic"))
+	inboundGroupBody.SetAttributeValue("from_port", cty.NumberIntVal(0))
+	inboundGroupBody.SetAttributeValue("to_port", cty.NumberIntVal(0))
+	inboundGroupBody.SetAttributeValue("protocol", cty.StringVal("-1"))
+	ipList = []cty.Value{}
+	ipList = append(ipList, cty.StringVal("0.0.0.0/0"))
+	inboundGroupBody.SetAttributeValue("cidr_blocks", cty.ListVal(ipList))
+}
+
+func setSecurityGroupRule(rootBody *hclwrite.Body, ipAddress, sgID string, ipInTCP, ipInHTTP bool) {
```

Comment:
We're going to need extensive documentation on exactly what we're setting up for the customer. We might want to start it now.